iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/function_ref.inl

```diff
@@ -15,15 +15,34 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#ifndef IOX_HOOFS_CXX_FUNCTION_REF_INL
+#define IOX_HOOFS_CXX_FUNCTION_REF_INL
+
+#include "iceoryx_hoofs/cxx/function_ref.hpp"
+#include "iceoryx_hoofs/cxx/requires.hpp"
+
+#include <memory>
+
 namespace iox
 {
 namespace cxx
 {
 template <class ReturnType, class... ArgTypes>
 template <typename CallableType, typename>
+// AXIVION Next Line AutosarC++19_03-A12.1.2 : Members are initialized in the same manner, NSDMI with nullptr is
```

Comment:
In what manner are the members initialized? What does NSDMI mean?

Reply:
@elfenpiff https://www.giybf.com :upside_down_face:

@MatthiasKillat Correct, I switchted it to `Next Construct`.

The rule just says either init via the `: m_foo(foo)` or inline via `Type m_foo{defaultValue}`, but don't mix things as you might forget it in another c'tor. However, doing it explicitly in both ways, is the best IMHO.